src/components/MAPI/workernodes/WorkerNodesNodePoolItem.tsx

```diff
@@ -74,8 +84,46 @@ const WorkerNodesNodePoolItem: React.FC<IWorkerNodesNodePoolItemProps> = ({
 
   const isScalingInProgress = scaling && scaling.desired !== scaling.current;
 
+  const viewAndEditNameRef = useRef<
+    { activateEditMode: () => boolean } & HTMLElement
+  >(null);
+
   const [isEditingDescription, setIsEditingDescription] = useState(false);
 
+  const onStartEditingDescription = () => {
+    viewAndEditNameRef.current?.activateEditMode();
+  };
```

Comment:
Don't really like this design, but let's go with it, to not break the current usage for GS API node pools.